var/spack/repos/builtin/packages/buddy/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import platform
+
+
+class Buddy(AutotoolsPackage):
+    """A Binary Decision Diagram library."""
+
+    homepage = "https://sourceforge.net/projects/buddy/"
+    url      = "https://sourceforge.net/projects/buddy/files/buddy/BuDDy%202.4/buddy-2.4.tar.gz"
+    list_url = url
```

Comment:
```suggestion
    list_url = "https://sourceforge.net/projects/buddy/files/buddy"
```

Reply:
fixed in https://github.com/spack/spack/pull/16166/commits/d1db50ac0ebf56cdc4df55067447a44c54499ac3